src/Psl/Arr/random.php

```diff
@@ -21,12 +23,13 @@
  */
 function random(array $values)
```

Comment:
Is it the goal to remove the `Arr\` namespace at some point?
Because this one, and some others, doesn't have an alternative version.
Maybe better to create/use a Iter\random that can operate on both vector and dict?

Now that I am typing this, I notice this PR adds vec only. So that move is probably a next stage :)

Reply:
exactly
- add vec\
- add dict\
- completely deprecate arr\ in favor of iter\ that can operate on all iterables
- add feature in the psalm plugin that allows Iter\count($array) to be called in a pure context, since it's not mutating anything